api/types.go

```diff
@@ -734,6 +734,17 @@ type Pin struct {
 	Reference cid.Cid
 }
 
+// PinSerialWithPath is a wrapper for holding pin serial and path of the content
+type PinSerialWithPath struct {
```

Comment:
I just realize that `PinPath` as it is, is already serializable. There is no need of `PinPathSerial` or anything like it.

Reply:
No, this PinPath should not have PinSerial inside, only PinOptions. And it should be embedded not a field in it.

Reply:
https://github.com/ipfs/ipfs-cluster/pull/634#discussion_r251288408